### PR TITLE
Translated faq/configure-a-booster-for-installation.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>

### CODE REVIEW

**Comment by Patrick-LuoYu** [comment]
\assign @Sherlock113

File: content/zh/docs/installing-on-kubernetes/faq/configure-a-booster-for-installation/为安装配置加速器.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+description: '如何为安装配置加速器'
+linkTitle: "为安装配置加速器"
+weight: 3610
+---
+
+如果您无法从 `dockerhub.io` 下载镜像，强烈建议您预先配置仓库的镜像地址（即加速器）以加快下载速度。您可以参考[ Docker 官方文档](https://docs.docker.com/registry/recipes/mirror/#configure-the-docker-daemon)，或执行以下步骤。
+
+## 获取加速器地址
+
+您需要获取仓库的一个镜像地址以配置加速器。 以下示例介绍如何从阿里云获取加速器地址。
```
Comment (Sherlock113) [inline content/zh/docs/installing-on-kubernetes/faq/configure-a-booster-for-installation/为安装配置加速器.md:13]: Delete the space between "...以配置加速器。" and "以下示例介绍..."

File: content/zh/docs/installing-on-kubernetes/faq/configure-a-booster-for-installation/为安装配置加速器.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+   ```json
+   {
+     "registry-mirrors": ["https://<my-docker-mirror-host>"]
+   }
+   ```
+
+   {{< notice note >}}
+
+    请将命令中的地址替换成您实际的加速器地址。
```
Comment (Sherlock113) [inline content/zh/docs/installing-on-kubernetes/faq/configure-a-booster-for-installation/为安装配置加速器.md:59]: Delete all the space before this sentence. There is sth wrong with the format.

image

File: content/zh/docs/installing-on-kubernetes/faq/configure-a-booster-for-installation/为安装配置加速器.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+       registryMirrors: [] # For users who need to speed up downloads
+       insecureRegistries: [] # Set an address of insecure image registry. See https://docs.docker.com/registry/insecure/
+       privateRegistry: "" # Configure a private image registry for air-gapped installation (e.g. docker local registry or Harbor)
+   ```
+
+2. 在 `registryMirrors` 处填入仓库的镜像地址并保存文件。关于安装过程的更多信息，请参见[多节点安装](../../../installing-on-linux/introduction/multioverview/)。
+
+{{< notice note >}}
+
+[在 Linux 上安装 All-in-one KubeSphere](../../../quick-start/all-in-one-on-linux/) 不需要 `config-sample.yaml` 文件。该模式下请采用第一种方法进行配置。
```
Comment (Sherlock113) [inline content/zh/docs/installing-on-kubernetes/faq/configure-a-booster-for-installation/为安装配置加速器.md:88]: 在 Linux 上安装 All-in-one KubeSphere
->
在 Linux 上通过 All-in-one 模式安装 KubeSphere

File: content/zh/docs/installing-on-kubernetes/faq/configure-a-booster-for-installation/为安装配置加速器.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
+---
```
Comment (Sherlock113) [inline content/zh/docs/installing-on-kubernetes/faq/configure-a-booster-for-installation/为安装配置加速器.md:1]: This is not the correct location of the file. The file should be in docs/installing-on-linux/faq/configure-booster/

**Review by Sherlock113 — COMMENTED** [review]
Have a look at my comments and relocate the file. Thanks.

**Pull request closed without merging**